backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -43,6 +43,55 @@
 #include "graphics/surface.h"
 #include "gui/EventRecorder.h"
 
+#ifdef PSP2
```

Comment:
Would it be possible to overload the SurfaceSdlGraphicsManager class instead of these #ifdefs?